### PR TITLE
changed icons

### PR DESCRIPTION
**Pull request by rohilpinto**

### CODE REVIEW

**Review by ivanms1 — APPROVED** [review]
Great job :+1:

**Review by Raj2631 — APPROVED** [review]
Good work

File: client/src/pages/Register/Register.js [hunk 1]
```diff
@@ -46,11 +46,11 @@ function Register() {
...
           },
         });
-
+        
         setRedirect(true);
-        // redirect with the id from the response
+        
       } catch (error) {
-        // display error
+
```
Comment (Raj2631) [inline client/src/pages/Register/Register.js]: add this comment back again

**Review by Raj2631 — CHANGES_REQUESTED** [review]

File: client/src/pages/Register/Register.js (hunk 1 above)
Comment (rohilpinto) [reply client/src/pages/Register/Register.js]: Done

**Review by Raj2631 — APPROVED** [review]

**Review by revengemiroz — APPROVED** [review]

**Comment by rohilpinto** [comment]
Changed Icons

File: client/src/components/Cardv2/style.js [hunk 2]
```diff
@@ -78,7 +86,13 @@ export const CardContainerInner = styled.div`
     height: 100%;
     justify-content: space-around;
     align-items: center;
+
+    & button {
+      /* transform: translateY(25px) */
```
Comment (ivanms1) [inline client/src/components/Cardv2/style.js]: remove please

File: client/src/pages/Register/Register.js [hunk 3]
```diff
@@ -46,11 +46,11 @@ function Register() {
...
           },
         });
-
+        
         setRedirect(true);
-        // redirect with the id from the response
+        
       } catch (error) {
-        // display error
+        // console.log(consoled)
```
Comment (ivanms1) [inline client/src/pages/Register/Register.js]: remove please

File: client/src/components/Cardv2/style.js (hunk 2 above)
Comment (rohilpinto) [reply client/src/components/Cardv2/style.js]: done

**Pull request closed without merging**